CHANGELOG.md

```diff
@@ -7,63 +7,72 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 NOTE: As semantic versioning states all 0.y.z releases can contain breaking changes in API (flags, grpc API, any backward compatibility)
 
-We use *breaking* word for marking changes that are not backward compatible (relates only to v0.y.z releases.)
+We use *breaking :warning:* word for marking changes that are not backward compatible (relates only to v0.y.z releases.)
```

Comment:
Nit: `word` here doesn't make sense. It would either need an article (the), eg `we use the words ... to mark...` OR be removed `we use ... to mark...`